### PR TITLE
Handle field placeholder when labels disabled

### PR DESCRIPTION
**Pull request by janowsiany**
Solves bug raised in #421

### CODE REVIEW

**Comment by janowsiany** [comment]
Should i put example of handling placeholder with GraphQLBridge extras?

File: packages/uniforms/src/GraphQLBridge.js [hunk 1]
```diff
@@ -137,6 +137,12 @@ export default class GraphQLBridge extends Bridge {
             ready.label = '';
         }
 
+        if (props.placeholder === true && extra.placeholder) {
+            ready.placeholder = extra.placeholder;
+        } else if (props.placeholder !== undefined && !props.placeholder) {
```
Comment (radekmie) [inline packages/uniforms/src/GraphQLBridge.js]: I'd rather check for `false` and `null` explicitly here.

File: packages/uniforms/__tests__/GraphQLBridge.js [hunk 2]
```diff
@@ -239,6 +244,34 @@ describe('GraphQLBridge', () => {
...
+                placeholder: 'Post ID',
+                required: true,
+                allowedValues: [1, 2, 3]
+            });
+        });
+
+        it('works with placeholder (falsy)', () => {
+            expect(bridgeI.getProps('id', {placeholder: null})).toEqual({
+                label: 'Post ID',
+                placeholder: '',
```
Comment (radekmie) [inline packages/uniforms/__tests__/GraphQLBridge.js]: Test for `placeholder === true` and `extra.placeholder === undefined` is missing.

File: packages/uniforms/src/JSONSchemaBridge.js [hunk 3]
```diff
@@ -144,19 +144,22 @@ export default class JSONSchemaBridge extends Bridge {
...
 
+        const fieldNameCapitalized = fieldName.charAt(0).toUpperCase() + fieldName.slice(1).toLowerCase();
+
         const ready = {
             allowedValues,
             ...(fieldType === 'number' ? {decimal: true} : {}),
             options: opts || options,
-            label: label === true ? fieldName.charAt(0).toUpperCase() + fieldName.slice(1).toLowerCase() : label || '',
+            label: label === true ? fieldNameCapitalized : label || '',
+            placeholder: placeholder === true ? fieldNameCapitalized : placeholder || undefined,
```
Comment (radekmie) [inline packages/uniforms/src/JSONSchemaBridge.js]: There's no need for `|| undefined`.

File: packages/uniforms/src/GraphQLBridge.js [hunk 4]
```diff
@@ -139,7 +139,7 @@ export default class GraphQLBridge extends Bridge {
 
         if (props.placeholder === true && extra.placeholder) {
             ready.placeholder = extra.placeholder;
-        } else if (props.placeholder !== undefined && !props.placeholder) {
+        } else if (props.placeholder !== undefined && (props.placeholder === false || props.placeholder === null)) {
```
Comment (radekmie) [inline packages/uniforms/src/GraphQLBridge.js]: Now the left part of `&&` is not needed. Please remove it and we are all set.

**Pull request merged**